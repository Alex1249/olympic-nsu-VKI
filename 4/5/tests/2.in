32 21
8 -642 497 921 -810 513 -647 933 -275 554 354 -126 -578 -852 726 116 -702 400 756 860 472 485 -254 -622 678 -934 -333 198 466 82 604 79
1 858
1 620
0 658
0 275
0 630
1 313
0 795
1 460
1 508
0 45
0 168
1 134
0 951
0 94
1 514
1 874
1 20
1 678
1 987
0 421
0 163
