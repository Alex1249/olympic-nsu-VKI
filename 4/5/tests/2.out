2056 -262786 497 921 -810 513 -647 933 -4371 618 354 -126 -578 -67109716 525014 124 -702 400 756 524636 472 33554917 -254 -622 678 -934 -333 2097350 466 82 604 79 
